Gruntfile.js

```diff
@@ -321,19 +330,15 @@ module.exports = function(grunt) {
       },
       'setup-admin': {
         cmd:
-          ` curl -X PUT ${couchConfig.withPath('_node/' + COUCH_NODE_NAME + '/_config/admins/admin')} -d '"${couchConfig.password}"'` +
-          ` && curl -X PUT --data '"true"' ${couchConfig.withPath('_node/' + COUCH_NODE_NAME + '/_config/chttpd/require_valid_user')}` +
-          ` && curl -X PUT --data '"4294967296"' ${couchConfig.withPath('_node/' + COUCH_NODE_NAME + '/_config/httpd/max_http_request_size')}` +
+          ` curl -X PUT ${couchConfig.withPathNoAuth(`_node/_local/_config/admins/${couchConfig.username}`)} -d '"${couchConfig.password}"'` +
+          ` && curl -X PUT --data '"true"' ${couchConfig.withPathNoAuth('_node/_local/_config/chttpd/require_valid_user')}` +
+          ` && curl -X PUT --data '"4294967296"' ${couchConfig.withPath('_node/_local/_config/httpd/max_http_request_size')}` +
```

Comment:
As above, this is good for now, but for 4.0 we'll be developing and testing against the cht couch image that does this out of the box, yeah?

Reply:
There's a section in the Development.md that specifically talks about installing Couch separately. That's the only case this should run. 
I didn't want this PR to do a readme overhaul.